container/rules/docker_toolchains.bzl

```diff
@@ -269,6 +234,81 @@ def _toolchain_container_impl(ctx):
       https://github.com/bazelbuild/rules_docker#container_image
     """
 
+    download_pkgs_output_tar = ctx.attr.name + "-download_pkgs_output_tar.tar"
+    download_pkgs_output_script = ctx.attr.name + "-download_pkgs_output_script.sh"
+
+    debian_output = _debian_pkg_tar_impl(ctx, download_pkgs_output_tar, download_pkgs_output_script)
```

Comment:
why did you decide to factor out the downloading of packages from inside _language_tool_layer_impl to here?

Reply:
This new rule can be used separately to create a tarball of debian packages in a BUILD file. 
```toolchain_container``` rule can also use the implementation of this rule.